src/kotlin/template.handlebars

```diff
@@ -49,7 +49,6 @@ class {{{apiClassName}}} : GBIORxWebServices() {
         {{/if}}
         @JvmSuppressWildcards
         fun {{{name}}}(
-            @HeaderMap headerMap: Map<String, String>{{#if params.length}},{{/if}}
```

Comment:
@noterB Hi Breton, just a general question, where are these parameters passed from? Thanks.

Reply:
They were passed in from each of the web service "query" objects.  (A poorly named convention imposed on us many years ago.)